### PR TITLE
changes to readme.md, adding examples, clarifications etc

### PR DESCRIPTION
**Pull request by Ths2-9Y-LqJt6**

### CODE REVIEW

**Comment by Ths2-9Y-LqJt6** [comment]
@melalj  - I liked the examples I added to each example, but it got a little verbose.  As well, I found some missing elements in the docs which I added to try and get a complete data set.  

Let's chat!

File: README.md [hunk 1]
```diff
@@ -30,13 +35,13 @@ MapTable is available on cdnjs.com. Remember though, cool kids concatenate their
...
+If you're running [Bower](http://bower.io/), you can install maptable like this:
+
 ```shell
 bower install --save maptable
 ```
 Notable files are: `build/maptable.min.js` `build/maptable.css`
 
-## Usage
-
 ## Naming conventions
```
Comment (melalj) [inline README.md]: Should be ###

File: README.md [hunk 2]
```diff
@@ -73,18 +78,15 @@ If you want to attach the data extremums to the value of an attribute. For examp
 }
 ```
 
-## Initiate MapTable
+## Adding a map
```
Comment (melalj) [inline README.md]: We can initiate MapTable without adding a map

File: README.md (hunk 1 above)
Comment (Ths2-9Y-LqJt6) [inline README.md]: Hmmm - I tweaked it to flatten it out a bit - see  ae6dbb9

File: README.md (hunk 2 above)
Comment (Ths2-9Y-LqJt6) [inline README.md]: yup yup - fixed in b2d65e2ec94a46737db2c19e2e879ba8677f647f

**Comment by Ths2-9Y-LqJt6** [comment]
@melalj  - when I try and use `.json('https://www.pch.net/ixp/locations')` now I get an error: 

`SyntaxError: JSON.parse: unexpected end of data at line 1 column 1 of the JSON data`

Only using the newly committed `ixp.csv` file seems to work with out error.  Can you verify this is a bug or is endemic to my dev environ?

**Comment by melalj** [comment]
@Ths2-9Y-LqJt6 I cannot reproduce this error. Maybe it's due to your `/etc/hosts` configurations.
Next time you get it, open the inspector on the network tab and  see what's inside the json file that you requested.

**Comment by melalj** [comment]
Merging this, thanks ;)

**Pull request merged**